2023-04-20 00:04:40,564 INFO    MainThread:8641 [wandb_setup.py:_flush():76] Configure stats pid to 8641
2023-04-20 00:04:40,565 INFO    MainThread:8641 [wandb_setup.py:_flush():76] Loading settings from /Users/malujie/.config/wandb/settings
2023-04-20 00:04:40,565 INFO    MainThread:8641 [wandb_setup.py:_flush():76] Loading settings from /Users/Uni Course/IML/IML_A3/wandb/settings
2023-04-20 00:04:40,565 INFO    MainThread:8641 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-04-20 00:04:40,566 INFO    MainThread:8641 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-04-20 00:04:40,566 INFO    MainThread:8641 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2023-04-20 00:04:40,566 INFO    MainThread:8641 [wandb_init.py:_log_setup():507] Logging user logs to /Users/Uni Course/IML/IML_A3/wandb/run-20230420_000440-o3ttn2dv/logs/debug.log
2023-04-20 00:04:40,567 INFO    MainThread:8641 [wandb_init.py:_log_setup():508] Logging internal logs to /Users/Uni Course/IML/IML_A3/wandb/run-20230420_000440-o3ttn2dv/logs/debug-internal.log
2023-04-20 00:04:40,567 INFO    MainThread:8641 [wandb_init.py:init():547] calling init triggers
2023-04-20 00:04:40,568 INFO    MainThread:8641 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-04-20 00:04:40,568 INFO    MainThread:8641 [wandb_init.py:init():595] starting backend
2023-04-20 00:04:40,569 INFO    MainThread:8641 [wandb_init.py:init():599] setting up manager
2023-04-20 00:04:40,571 INFO    MainThread:8641 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-04-20 00:04:40,576 INFO    MainThread:8641 [wandb_init.py:init():605] backend started and connected
2023-04-20 00:04:40,591 INFO    MainThread:8641 [wandb_run.py:_label_probe_notebook():1237] probe notebook
2023-04-20 00:04:40,591 INFO    MainThread:8641 [wandb_run.py:_label_probe_notebook():1247] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-04-20 00:04:40,592 INFO    MainThread:8641 [wandb_init.py:init():695] updated telemetry
2023-04-20 00:04:40,619 INFO    MainThread:8641 [wandb_init.py:init():732] communicating run to backend with 60.0 second timeout
2023-04-20 00:05:40,834 ERROR   MainThread:8641 [wandb_init.py:init():758] encountered error: Run initialization has timed out after 60.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2023-04-20 00:05:40,930 ERROR   MainThread:8641 [wandb_init.py:init():1163] Run initialization has timed out after 60.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1145, in init
    run = wi.init()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 764, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 60.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
